SPECS/rust/rust.spec

```diff
@@ -85,23 +86,18 @@ tar -xf %{SOURCE1} --no-same-owner
 popd
 %autosetup -p1 -n rustc-%{version}-src
 
-# Rust doesn't recognize our .tar.gz bootstrap files when XZ support is enabled
-# This causes stage 0 bootstrap to look online for sources
-# So, we remove XZ support detection in the bootstrap program
-sed -i "s/tarball_suffix = '.tar.xz' if support_xz() else '.tar.gz'/tarball_suffix = '.tar.gz'/g" src/bootstrap/bootstrap.py
-
 # Setup build/cache directory
 BUILD_CACHE_DIR="build/cache/%{release_date}"
 mkdir -pv "$BUILD_CACHE_DIR"
 %ifarch x86_64
-mv %{SOURCE2} "$BUILD_CACHE_DIR"
-mv %{SOURCE3} "$BUILD_CACHE_DIR"
-mv %{SOURCE4} "$BUILD_CACHE_DIR"
+cp %{SOURCE2} "$BUILD_CACHE_DIR"
```

Comment:
why cp instead of mv?

Reply:
`mv` means you can't re-run a build since it effectively deletes the source files from `/usr/src/mariner/SOURCES`. You have to re-install the srpm every time you want to build.